pkgs/applications/video/mkvtoolnix/default.nix

```diff
@@ -89,7 +88,7 @@ mkDerivation rec {
     xdg-utils
     zlib
   ]
-  ++ optional withGUI cmark
+  ++ optionals withGUI [ cmark wrapQtAppsHook ]
```

Comment:
There is already a `wrapQtAppsHook` properly added to the `nativeBuildInputs` above.

Reply:
My bad, I thought I was adding it to `nativeBuildInputs`. I'll fix it.